docs/index.rst

```diff
@@ -1,4 +1,5 @@
 .. mdinclude:: ../README.md
+.. mdinclude:: ..Flatpak_Build.md
```

Comment:
InputError: [Errno 2] No such file or directory: '../../../../../../checkouts/latest/docs/..Flatpak_Build.md'.
Raw: https://readthedocs.org/api/v2/build/14080458.txt

Reply:
I meant something like this: https://pybitmessage-test.readthedocs.io/en/latest/#build-instructions
Did it with these changes: 0bedc5d